.github/workflows/ci.yml

```diff
@@ -157,7 +157,7 @@ jobs:
 
       - name: run redis tests
         if: matrix.test-mode == 'defaults'
-        run: gotestsum --format short-verbose -- -p 1 -run TestSeqCoordinator -tags redistest ./arbnode/... ./system_tests/... -coverprofile=coverage-redis.txt -covermode=atomic -coverpkg=./...
+        run: gotestsum --format short-verbose -- -p 1 -run TestRedis -tags redistest ./arbnode/... ./system_tests/... -coverprofile=coverage-redis.txt -covermode=atomic -coverpkg=./...
```

Comment:
I think there's a couple tests which aren't redis-only but will enable redis functionality `if redisUrl != ""`, like the batch poster parallel test. We should tag those as redis too.

Reply:
Having all redis tests be named TestRedis* still seems simpler. adding this test as well.